test/clojure-mode-font-lock-test.el

```diff
@@ -780,13 +780,13 @@ DESCRIPTION is the description of the spec."
      (2 4 font-lock-type-face)
      (5 5 nil)
      (6 9 font-lock-keyword-face)
-     (11 13 font-lock-function-name-face))
+     (11 13 nil))
```

Comment:
I guess this example should be updated to use `clojure-core`, otherwise it's kind of pointless.

Reply:
Yeah, makes sense. Done